explorer/rest/tests/test_rest.py

```diff
@@ -59,15 +59,21 @@ def client(app):  # pylint: disable=redefined-outer-name
 # endregion
 
 
+def _assert_status_code_and_headers(response, expected_status_code):
+	# Assert:
+	assert expected_status_code == response.status_code
+	assert response.headers['Access-Control-Allow-Origin'] == '*'
+
+
 # region /block/<height>
 
 def _assert_get_api_nem_block_by_height(client, height, expected_status_code, expected_result):  # pylint: disable=redefined-outer-name
 	# Act:
 	response = client.get(f'/api/nem/block/{height}')
 
 	# Assert:
-	assert expected_status_code == response.status_code
 	assert expected_result == response.json
+	_assert_status_code_and_headers(response, expected_status_code)
```

Comment:
generally would swap order because `_assert_status_code_and_headers` is testing common part and should be in most tests and then more specific checks follow

Reply:
make sense 👍🏼